### DISCUSSION TITLE
[help]添加章节/书签

### DISCUSSION THREAD

**Discussion by sun2ot** [hooke007/mpv_PlayKit#408] [Q&A 问答] [answered]
请问有无类似potplayer中按p添加书签的脚本？有一些会议记录之类的长视频需要借助章节/标签实现快速跳转，否则来回切换太麻烦了。

我已在mpv的文档中检索 `chapter` or `bookmark` 之类的关键词，没有找到相关的添加手段。

**Comment by verygoodlee** [comment, marked as answer]
这些都可以试试
https://github.com/shinchiro/mpv-createchapter
https://github.com/dyphire/mpv-scripts?tab=readme-ov-file#chapter-make-readlua
https://github.com/mar04/chapters_for_mpv

**Reply by sun2ot** [reply]
谢谢，我尝试一下

**Reply by hooke007** [reply]
。。。你是怎么做到搜不到这些的 https://github.com/mpv-player/mpv/wiki/User-Scripts
